src/agents/OIDCCredentialAgent.js

```diff
@@ -9,14 +8,11 @@ import fetch from '../fetch';
  * with many Client instances.
  */
 export default class OIDCCredentialAgent {
-  constructor({
-    accessToken,
-    url = withRootUrl(process.env.TASKCLUSTER_ROOT_URL).api(
-      'login',
-      'v1',
-      'oidc-credentials'
-    ),
-  }) {
+  constructor({ accessToken, url }) {
```

Comment:
Would it be easier to use rootUrl here?

Reply:
My thought was that the consumer here would use lib-urls to generate a login URL, since in the current scenario the path cannot be overridden. Maybe that's ok.